src/fidesops/graph/config.py

```diff
@@ -223,6 +225,26 @@ def identities(self) -> Dict[str, Tuple[str, ...]]:
         flds_w_ident = filter(lambda f: f.identity, self.fields)
         return {f.name: f.identity for f in flds_w_ident}
 
+    @property
+    def categories_to_fields(self) -> Dict[str, List]:
```

Comment:
Minor nit - I think it's be a bit clearer for the function name to be consistent with the fact that it's a property rather than a verb, e.g something like "fields_by_category"

Reply:
i agree, that's a better name, thanks!